cookbook/integrations/flytekit_plugins/mlflow_example/README.rst

```diff
@@ -0,0 +1,42 @@
+.. mlflow:
+
+MLflow
+======
+
+.. tags:: Integration, Data, Metrics, Intermediate
+
+The MLflow Tracking component is an API and UI for logging parameters,
+code versions, metrics, and output files when running your machine learning code and for later visualizing the results
+
+`MLflow <https://mlflow.org/>`__ is used by Flyte to automatically log the task's metrics and parameters to either Flyte Deck or MLflow server.
+
+.. figure:: https://user-images.githubusercontent.com/37936015/200655711-8eb54757-cc08-4591-8f44-787cc4b0af66.png
+  :alt: Model Metrics
+  :class: with-shadow
+
+.. figure:: https://user-images.githubusercontent.com/37936015/200655752-fddfd0cd-26f2-4ccf-914a-08044c7c4dae.png
+  :alt: Model Parameters
+  :class: with-shadow
+
+To log the metrics and parameters to Flyte deck, add `@mlflow_autolog` to the task. For example
+
+.. code:: python
+
+    @task(disable_deck=False)
+    @mlflow_autolog(framework=mlflow.keras)
+    def train_model(epochs: int):
+    ...
+
+To log the metric and parameters to a remote mlflow server, add default env (`MLFLOW_TRACKING_URI <https://mlflow.org/docs/latest/tracking.html#logging-to-a-tracking-server>`__) to the flytepropeller config map.
```

Comment:
can we also provide instructions on how to apply to this config to the Flyte cluster?